Makefile

```diff
@@ -0,0 +1,18 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
I really want to avoid a top-level Makefile. This is a Python project, and this may even confuse build system autodetection for distributions/packagers, let alone humans. If there is no pyproject.toml moral equivalent to npm package.json's `"scripts:" {}`, then perhaps a `./build-sphinx.sh`?

Reply:
The Makefile is just a helper for us (dbusmock devs), even the CI just calls `sphinx-build -M html doc/ doc/` directly. I've dropped it now.

I'm not sure how much value a locally built sphinx-build is over building it online and referring to readthedocs. So I'm not sure integrating it into `pyproject.toml` is really worth it? If anything we'll just have to spend time debugging it where it doesn't work :)